monkestation/code/modules/antagonists/bloodling/abilities/devour.dm

```diff
@@ -0,0 +1,56 @@
+/datum/action/cooldown/mob_cooldown/bloodling/devour
```

Comment:
File thats not supposed to be here, and is from another branch
Do not merge until dis is deleted

Reply:
Yep, snuck in with an upstream merge. I'll remove it.